klippy/kinematics/winch.py

```diff
@@ -47,7 +47,11 @@ def check_move(self, move):
         pass
     def get_status(self, eventtime):
         # XXX - homed_checks and rail limits not implemented
-        return {'homed_axes': 'xyz'}
+        return {
+            'homed_axes': 'xyz',
+            'axis_minimum': {'x': -200, 'y': -200, 'z': -200},
+            'axis_maximum': {'x': 200, 'y': 200, 'z': 200}
```

Comment:
There's no enforcement of boundaries on winch.  However, a good approximation would be to limit to the minimum/maximum x,y,z of `self.anchors`.  For example, `min_x = min([a[0] for a in self.anchors])`.

-Kevin